pkgs/by-name/co/cosmic-term/package.nix

```diff
@@ -71,23 +69,33 @@ rustPlatform.buildRustPackage rec {
     (placeholder "out")
     "--set"
     "bin-src"
-    "target/${
-      rust.lib.toRustTargetSpecShort stdenv.hostPlatform
-    }/release/cosmic-term"
+    "target/${stdenv.hostPlatform.rust.cargoShortTarget}/release/cosmic-term"
+  ];
+
+  # Force linking to libEGL, which is always dlopen()ed, and to
+  # libwayland-client, which is always dlopen()ed except by the
+  # obscure winit backend.
+  RUSTFLAGS = map (a: "-C link-arg=${a}") [
+    "-Wl,--push-state,--no-as-needed"
+    "-lEGL"
+    "-lwayland-client"
+    "-Wl,--pop-state"
   ];
 
   # LD_LIBRARY_PATH can be removed once tiny-xlib is bumped above 0.2.2
   postInstall = ''
     wrapProgram "$out/bin/${pname}" \
       --suffix XDG_DATA_DIRS : "${cosmic-icons}/share" \
-      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ xorg.libX11 xorg.libXcursor xorg.libXrandr xorg.libXi wayland libxkbcommon ]}
+      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [
+        xorg.libX11 xorg.libXcursor xorg.libXi xorg.libXrandr libxkbcommon vulkan-loader mesa.drivers
+      ]}
   '';
 
   meta = with lib; {
     homepage = "https://github.com/pop-os/cosmic-term";
     description = "Terminal for the COSMIC Desktop Environment";
     license = licenses.gpl3Only;
-    maintainers = with maintainers; [ ahoneybun ];
+    maintainers = with maintainers; [ ahoneybun nyanbinary ];
```

Comment:
Could you move this change to a commit of it's own, as well as anything you don't regard as required for the version bump?

Reply:
Is there a reason? Seems silly to make a whole PR just for correcting the name.

Reply:
separate commit,  same PR

Reply:
That makes sense, sorry I got confused there. I'll do that.

Reply:
Should be good now.

Reply:
The commit name for the bump should be ``0-unstable-2024-01-19``

Reply:
Please prefix the commit message with `cosmic-term:`, so it's in line with the [commit conventions](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#commit-conventions).

Reply:
E.g.: ``cosmic-term: fix co-maintainer name``

Reply:
Should be good now.

Reply:
I'm sorry to still be nitpicking about this, but I think keeping git history clean is somewhat important considering the size of the project.

I don't think there's good reason to add `nyabinary` in the first commit, just to fix the name to `nyanbinary` in the second commit. It would be better to create a commit named `cosmic-term: add nyanbinary to maintainers`, and drop the change altogether for the first commit.

Secondly, there seems to be a change to the version number in the `fix co-maintainer name` commit without any new rev or hash. Is this intended to be a new version, or did you miss on the first date? Either way, I think this should be moved back into the first commit.

Feel free to ask if you need any help with git fu.

Reply:
Alright it **should** look good now!

Reply:
Excellent, thank you!